bluez-qt/bluetoothdevicemodel.cpp

```diff
@@ -228,21 +211,11 @@ void BluetoothDevicesModel::adapterRemoved(QDBusObjectPath)
 {
 	QDBusObjectPath adapterpath = manager->DefaultAdapter();
```

Comment:
This is a blocking D-Bus call. Why couldn't it be something like this:
```
    if (adapter && adapter->path() == path.path()) {
```
where path is the adapterRemoved argument (which currently unused).